package.json

```diff
@@ -3,23 +3,26 @@
   "version": "1.0.0",
   "description": "elastic.io webhook component",
   "scripts": {
-    "test": "./node_modules/jasmine-node/bin/jasmine-node --test-dir spec"
+    "test": "mocha spec --recursive"
   },
   "engines": {
     "node": "6.4.0"
   },
   "author": "elastic.io GmbH",
   "license": "BSD-2-Clause",
```

Comment:
Should be changed to: Apache License 2.0 as that what LICENSE file we have in this project